add Gnolang 101 (#15)
gnolang/awesome-gno: README.md

* [Quickstart Guide](https://test2.gno.land/r/boards:testboard/5) - How to start interacting with the blockchain.
* [A Beginner’s Guide to the Gnoland Testnet](https://medium.com/@onbloc/a-beginners-guide-to-the-gnoland-testnet-6fdc693a48f4) - A visual guide to creating a wallet and receiving $GNOTs on the testnet.

> * [GNOLAND 101](https://github.com/onbloc/gnolang-101) - A course designed for aspiring programmers who are looking to become a smart contract developer on Gnoland

Reviewer: ```suggestion
* [Gnolang 101](https://github.com/onbloc/gnolang-101) - A course designed for aspiring smart-contract developers on Gnoland.
* ```
